src/components/diagnosticReportView/DiagnosticReportCard.tsx

```diff
@@ -114,18 +118,32 @@ const DiagnosticReportCard: React.FC<DiagnosticReportCardProps> = ({
   const [currentTab, setCurrentTab] = useState<number>(0);
   const [hazardId, setHazardId] = useState<string>('');
   const [filteredMeta, setFilteredMeta] = useState<MetaArguments>([]);
+  const [regional, setRegional] = useState<boolean>(true);
 
   useEffect(() => {
     setCurrentImage(0);
   }, [automationTasks]);
 
   useEffect(() => {
     if (reportTab !== 0) setCurrentTab(reportTab ?? 0);
-  }, []);
+  }, [reportTab]);
 
   useEffect(() => {
     setReportTab && setReportTab(currentTab);
-  }, [currentTab]);
+  }, [setReportTab, currentTab]);
+
+  useEffect(() => {
+    if (automationTasks[currentImage]) {
+      const tvzValue = automationTasks[currentImage].inversion_solution?.meta?.filter(
```

Comment:
@benjamineac Just a heads-up that is where things can get fragile, since the API doesn't control the contents of meta. As @chrisdicaprio know its set by our Runzi configurations. 

For the public UI I think we'll want these implemented as API fields, making the client code cleaner, and everything more robust.